src/influxdb_utils.sh

```diff
@@ -6,24 +6,31 @@
 wait_for_influxdb_start(){
   # InfluxDB can take some time to start
   # Retry `influx ping` until we receive confirmation that it is up and running
+  # Influx commands need to be given the port of InfluxDB inside the container, which is always 8086 unless overridden inside the InfluxDB config
   CONTAINER_NAME=$1
   INFLUXDB_PORT=$2
   SERVER_PROTOCOL=$3
+  SKIP_TLS_VERIFY=$4
 
-  if [[ -z $CONTAINER_NAME || -z $INFLUXDB_PORT || -z $SERVER_PROTOCOL ]]; then
-    echo 'Container name, InfluxDB port, or server protocol was not provided when waiting for InfluxDB to start!'
+  if [[ -z $CONTAINER_NAME || -z $INFLUXDB_PORT || -z $SERVER_PROTOCOL || -z $SKIP_TLS_VERIFY ]]; then
+    echo 'Container name, InfluxDB port, server protocol, or skip TLS verify was not provided when waiting for InfluxDB to start!'
     exit 1
   fi
 
+  SKIP_TLS_VERIFY_ARG=""
+  if [ "$SKIP_TLS_VERIFY" == "true" ]; then
+    SKIP_TLS_VERIFY_ARG="--skip-verify"
+  fi
+
   CONTAINER_SETUP_STATUS=""
   RETRIES=0
   until [ "$CONTAINER_SETUP_STATUS" == "OK" ] || [ "$RETRIES" -eq 4 ]; do
     sleep 10
     echo "Attempt $RETRIES: Waiting until InfluxDB reports a status of OK..."
-    if [ $SERVER_PROTOCOL == "http" ]; then
-      CONTAINER_SETUP_STATUS=$(docker exec $CONTAINER_NAME influx ping) || true
-    elif [ $SERVER_PROTOCOL == "https" ]; then
-      CONTAINER_SETUP_STATUS=$(docker exec $CONTAINER_NAME influx ping --host "https://$CONTAINER_NAME:$INFLUXDB_PORT" --skip-verify) || true
+    if [ "$SERVER_PROTOCOL" == "http" ]; then
+      CONTAINER_SETUP_STATUS=$(docker exec "$CONTAINER_NAME" influx ping --host "http://$CONTAINER_NAME:8086") || true
```

Comment:
Why are we hard-coding 8086? Is it fixed?

Reply:
Yes - 8086 here is the port inside the container, which we map to a configurable port